package.json

```diff
@@ -25,6 +25,9 @@
 		"eslint-plugin-prettier": "^5.1.0",
 		"prettier": "^3.2.3"
 	},
+	"peerDependencies": {
+		"eslint": "^8.56.0"
```

Comment:
Should this be `>` rather than `^`, so that people _can_ use eslint 9 if they prefer?

Reply:
I didn't know this was a thing!